app/services/relocation_manager.py

```
@@ -263,21 +268,33 @@ def progress_callback(bytes_transferred: int):
-                raise Exception(msg)
+            # Perform the move — remote backends upload via their own API
+            if is_remote_backend:
+                success, error, storage_reference, _checksum = target_backend.freeze_file(
+                    source_path=current_file_path,
+                    relative_path=relative_path,
+                    location=target_location,
+                    operation_mode=OperationType.MOVE,
+                )
+                if not success:
+                    msg = f"File move failed: {error}"
+                    raise Exception(msg)
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)